include/ck_tile/ops/fmha/pipeline/block_fmha_bwd_pipeline_problem.hpp

```diff
@@ -24,7 +24,9 @@ template <typename QDataType_,
           typename BiasGradDataType_,
           typename BlockFmhaShape_,
           bool kIsGroupMode_,
+          bool kIsDeterministic_,
           typename FmhaMask_,
+          typename FmhaDropout_,
           typename Traits_>
```

Comment:
Suggest remove `Traits` from being as template of `Problem`,  add `Trait` as template of `Pipeline`,  actually only the kernel use `Traits` when creating the alignment and windows   `Trait` is a purely implementation specific, it is conceptually confusing to be part of `Problem`